step.yml

```diff
@@ -132,29 +158,41 @@ inputs:
   - public_install_page_url_map_format: "{{range $index, $element := .}}{{if $index}}|{{end}}{{$element.File}}=>{{$element.URL}}{{end}}"
     opts:
       title: Format for the BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP output
-      summary: You can change the format of the BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP output
+      summary: You can customize the format of the `BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP` output so that the next Step can render the output.
       description: |-
-        Specify the output pattern with golang template: [https://golang.org/pkg/text/template](https://golang.org/pkg/text/template)
+        Provide a language template description using [https://golang.org/pkg/text/template]([https://golang.org/pkg/text/template)
+        so that the **Deploy to Bitrise.io** Step can build the required custom output.
       is_required: true
       is_expand: false
   - permanent_download_url_map_format: "{{range $index, $element := .}}{{if $index}}|{{end}}{{$element.File}}=>{{$element.URL}}{{end}}"
     opts:
       title: Format for the BITRISE_PERMANENT_DOWNLOAD_URL_MAP output
-      summary: You can customize the format of the BITRISE_PERMANENT_DOWNLOAD_URL_MAP output so that it fits the next Step you wish to use the output with.
+      summary: You can customize the format of the `BITRISE_PERMANENT_DOWNLOAD_URL_MAP` output so that the next Step can render the output.
       description: |-
-        Customize the output pattern with this golang template: [https://golang.org/pkg/text/template](https://golang.org/pkg/text/template)
+        Provide a language template description using [https://golang.org/pkg/text/template]([https://golang.org/pkg/text/template)
+        so that the **Deploy to Bitrise.io** Step can build the required custom output for the permanent download URL. 
+        This URL can be shared in any communication channel and it will not expire.
```

Comment:
This line and next ones should go to the output. Also the output contains not a single URL but a set of urls for the different uploaded artifacts